mw-webapp/src/logic/usersTable/UsersTable.tsx

```diff
@@ -0,0 +1,45 @@
+import {ColumnDef, getCoreRowModel, useReactTable} from "@tanstack/react-table";
+import {Table} from "src/component/table/Table";
+import {UserPreview} from "src/model/businessModelPreview/UserPreview";
+
+/**
+ * Ways table props
+ */
+interface UsersTableProps {
+
+  /**
+   * Ways table data
+   */
+  data: UserPreview[];
+
+  /**
+   * Ways table columns type
+   * It's a bug of tanstack. We can't use type instead of any in this case.
+   * That because this type depends on type of each columns of table.
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Need to add link to github issue. Or add TODO comment for to add link

Reply:
Please send me a link to issue if you have it, i don't how it's called and where to find it

Reply:
I don't have a link

Reply:
I changed everything, i dont know why this comment was here instead of comment about bug